modules/sdp-simulcast/index.js

```diff
@@ -0,0 +1,510 @@
+/* Copyright @ 2016 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var transform = require('sdp-transform');
+var transformUtils = require('./transform-utils');
+var parseSsrcs = transformUtils.parseSsrcs;
+var writeSsrcs = transformUtils.writeSsrcs;
+
+//region Constants
+
+var DEFAULT_NUM_OF_LAYERS = 3;
+
+//endregion
+
+function getSsrcAttribute (mLine, ssrc, attributeName) {
+    return mLine
+        .ssrcs
+        .filter(function(ssrcInfo) { return ssrcInfo.id === ssrc; })
+        .filter(function(ssrcInfo) { return ssrcInfo.attribute === attributeName; })
+        .map(function(ssrcInfo) { return ssrcInfo.value; })[0];
+}
+
+//region Ctor
+
+function Simulcast(options) {
```

Comment:
Moved sdp-simulcast to the same repo to make working on it easier. It's easy to copy paste into the original repo and get the diff.